Newrelic/NameableInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Arxus\NewrelicMessengerBundle\Newrelic;
+
+interface NameableInterface
```

Comment:
Perhaps a more descriptive name would be better?
Maybe `NewrelicNamebleInterface`?

Reply:
I think the namespace is enought

Reply:
Maybe opt for ShouldHaveTransactionName or HasTransactionName?

Reply:
Nameable is more meaningful for me

Reply:
I still prefer `NamableTransactionInterface` or the likes.
Personally i'd even go as far to call it `NameableNewrelicTransactionInterface`.

I like descriptive names, without having to be forced to look at the namespace :|

Reply:
I updated it myself

Reply:
as you want